citylearn/building.py

```diff
@@ -1480,11 +1589,51 @@ def estimate_action_space(self) -> spaces.Box:
                     else:
                         raise Exception(f'Unknown action: {key}')
 
-                    limit = power/max(capacity, ZERO_DIVISION_PLACEHOLDER)
-                    
+                    limit = power / max(capacity, ZERO_DIVISION_PLACEHOLDER)
+
                 limit = min(limit, 1.0)
                 low_limit.append(-limit)
                 high_limit.append(limit)
+
```

Comment:
Do not commit the changes in the `estimate_action_space` method yet. There are recent changes I made that I don't think is captured here.

Reply:
WHat should i Do here ?

Reply:
Should I wait ?